src/app/modal/modal.component.html

```diff
@@ -6,18 +6,18 @@
                 <div class="modal-body">
                     <div class="row">
                         <div class="col">
-                        <input type="text" name="lastname" id="lastname" placeholder="Nom" formControlName="lastname">
+                        <input type="text" name="lastname" id="lastname" placeholder="Nom" formControlName="lastName">
                         </div>
                         <div class="col"> 
-                        <input type="text" name="firstname" id="firstname" placeholder="Prénom" formControlName="firstname">
+                        <input type="text" name="firstname" id="firstname" placeholder="Prénom" formControlName="firstName">
                         </div>
                     </div>
                     <div class="row">
                         <div class="col py-2">  
                             <label for="state"> Statut : </label>
-                            <select name="statut" id="statut" formControlName="statut">
+                            <select name="statut" id="statut" formControlName="status">
                                 <option value="INTERNE">INTERNE</option>
```

Comment:
interne ?

Reply:
Sorry, it's an forget of my test 'toLowerCase()'